### PR TITLE
Added image, list, table, form and button to index.html. Added <meta>

### PR DESCRIPTION
**Pull request by DariaRomanowska**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,12 +2,190 @@
...
     <title>Main Page</title>
   </head>
   <body>
-  <h1>Моя первая страница</h1>
-  <p>
-    <a href="./about">Об авторе</a>
-  </p>
+    <h1>Моя первая страница</h1>
+    <p>
+      <a href="./about">Об авторе</a>
```
Comment (5tronciy) [inline index.html]: Does it work in your browser?

File: index.html [hunk 2]
```diff
@@ -2,12 +2,190 @@
...
+        <tr>
+          <th align="left">&lt;tfoot&gt;</th>
+          <th align="left">Groups the foot content in a table</th>
+        </tr>
+      </tbody>
+    </table>
+    <form action="/" method="get">
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
```
Comment (5tronciy) [inline index.html]: `<p></p>` is for the text paragraph.
Block should be wrapped into `<div></div>`

File: index.html [hunk 3]
```diff
@@ -2,12 +2,190 @@
...
+          <label for="password">Password</label><br />
+          <input id="password" required type="password" name="password" />
+        </p>
+        <p>
+          <label for="programming">Your programming level</label><br />
+          <input
+            id="programming"
+            type="range"
+            name="level"
+            min="0"
```
Comment (5tronciy) [inline index.html]: 0 doesn't match the range 1-10.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
In general good. But you have ignored my remarks for the last PR.
Make commit with changes before merge this PR.

File: index.html (hunk 1 above)
Comment (DariaRomanowska) [reply index.html]: Yes, it works, but now I changed it to the full way

**Review by 5tronciy — APPROVED** [review]
Looks good.

**Pull request merged**